dev/gravityform-request-email.json

```diff
@@ -22,7 +22,7 @@
                 "inputs": null,
                 "inputType": "email",
                 "formId": 1,
-                "description": "Please enter your email address.",
```

Comment:
"we" is used in this description and "Webcom" is use in other descriptions. I would suggest we use Webcom instead of we in all descriptions.

Reply:
Yeah, we definitely need to at least specify who "we" are first--I've updated the description above the form in Dev so that this initial "we" is clarified.